tests/e2e/e2e_query_exec_test.go

```diff
@@ -0,0 +1,47 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	ccvtypes "github.com/cosmos/interchain-security/x/ccv/provider/types"
+
+	"github.com/cosmos/cosmos-sdk/client/flags"
+)
+
+func (s *IntegrationTestSuite) execQueryConsumerChains(
```

Comment:
Maybe also check that proper clients and channels got established?
The more complex details are checked in ICS

Reply:
Yeah great idea I'll add in a follow up PR